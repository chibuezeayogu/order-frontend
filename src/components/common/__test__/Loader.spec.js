import React from 'react'
import { mount } from 'enzyme'
import Loader from '../Loader'

describe('Loader Component', () => {
  it('Should render without crashing', () => {
    const wrapper = mount(<Loader />)
    expect(wrapper).toMatchSnapshot()
  })
})
