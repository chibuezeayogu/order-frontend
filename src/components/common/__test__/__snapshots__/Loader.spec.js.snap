// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component Should render without crashing 1`] = `
<Loader>
  <div
    className="text-center"
  >
    <div
      className="spinner-border"
      role="status"
    >
      <span
        className="visually-hidden"
      >
        Loading...
      </span>
    </div>
  </div>
</Loader>
`;
